marimo/_plugins/stateless/json.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2025 Marimo. All rights reserved.
+from __future__ import annotations
+
+from json import loads
+from typing import Optional
+
+from marimo._output.hypertext import Html
+from marimo._output.rich_help import mddoc
+from marimo._plugins.stateless import json_output
+
+
+@mddoc
+def json(
```

Comment:
you might nee to run `make py-snapshots` since we snapshot the public API and this updates it

Reply:
it's what I'm trying to find in case of updating docs! Thanks